spec/compiler/debug_spec.cr

```diff
@@ -0,0 +1,94 @@
+require "../../spec_helper"
+
+describe "Code gen: debug" do
+  it "runs" do
+    debug %(
+      (gdb) break foo
+      (gdb) run
+      (gdb) next
```

Comment:
This line is required because `break foo` seems to stop at `spec:1` (i.e. the `def` line). In #3476 this wasn't required, but it seems to be here.